lib/resque/scheduler/scheduling_extensions.rb

```diff
@@ -101,8 +109,10 @@ def all_schedules
 
       # clean the schedules as it exists in redis, useful for first setup?
       def clean_schedules(keys = non_persistent_schedules)
-        keys.each do |key|
-          remove_schedule(key)
+        redis.multi do
```

Comment:
why this? what problem does this line here solve? is that necessary? how is it related to the rest of this PR?